extensions/wikia/Blogs/BlogsHelper.class.php

```diff
@@ -163,12 +163,6 @@ public static function onTitleMoveComplete( Title $oOldTitle, Title $oNewTitle,
 	 * @return bool false if we're trying to move out of or into blog comment namespace, true otherwise
 	 */
 	public static function onAbortMove( Title $sourceTitle, Title $targetTitle, User $user, &$err, string $reason ): bool {
-		$blogsNS = [ NS_BLOG_ARTICLE, NS_BLOG_ARTICLE_TALK ];
-		if ( $sourceTitle->inNamespaces( $blogsNS ) && !$targetTitle->inNamespace( $sourceTitle->getNamespace() ) ) {
-			$err = wfMessage( 'immobile-source-namespace', $sourceTitle->getNsText() )->escaped();
-			return false;
-		}
-
```

Comment:
```
SUS-260: Prevent moving pages into User blog comment namespace, or moving comments out of this namespace
```

This comment needs to be updated to avoid confusion in the future.